tests/lava/utils/test_plots.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import unittest
+import numpy as np
+import matplotlib.pyplot as plt
+from lava.utils.plots import raster_plot
+
+np.random.seed(0)
+_SPIKES = np.random.randint(2, size=(10, 20))
```

Comment:
You could also set this up like this
```python
class TestInputValidation(unittest.TestCase):
    def setUp(self) -> None:
        """This will be executed to set up every test."""
        self.spikes = np.random.randint(2, size=(10, 20))

    def test_return_figure_on_valid_input(self) -> None:
        fig = raster_plot(self.spikes)
        ...
```

Reply:
Thanks for reviewing @mathisrichter! On the above: the main difference I can see would be that ```spikes``` would get a different value for each sub-test. Is there value in this or is this proposed change for improving the code style?

Reply:
This was rather a point on code style. This may be personal taste - I prefer such member variables over variables in the module scope.

If you want the same values for each test, you could set the seed in the `setUp` method as well.